base/ltoutput.dtx

```diff
@@ -959,11 +959,23 @@
     \fi
   \fi
   \newpage
-  \write\m@ne{}%
+%    \end{macrocode}
+%    This needs to be non-immediate even if we are already past the
```

Comment:
This next write needs to be . . . 

And: does this fact need further explanation?

Reply:
If you remember why it is needed @car222222 we can document it better. I know it is but ... (Leslie's documentation is up earlier in the file and it just says "part of a hack to make sure no \write's are lost")